src/ray/protobuf/serialization.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package ray.serialization;
+
+message PythonBuffer {
```

Comment:
Can you document the fields down here? Especially which buffer location "address" is relative to :)